Backend/Remora.Discord.API.Abstractions/API/Objects/Users/IPartialUser.cs

```diff
@@ -42,7 +42,7 @@ public interface IPartialUser
     Optional<ushort> Discriminator { get; }
 
     /// <inheritdoc cref="IUser.GlobalName" />
-    Optional<string?> GlobalName { get; }
+    // Optional<string?> GlobalName { get; }
```

Comment:
I'm not super keen on leaving code commented out like this - preferably, you'd leave an explanation at the class/interface level documentation instead in a `<remarks>` section.

Reply:
removed these and set the remarks with inheritdoc to IUser's remarks